src/test/java/DeadlineTest.java

```diff
@@ -0,0 +1,39 @@
+import Duke.task.Deadline;
+import Duke.task.Task;
+import org.junit.jupiter.api.Test;
+
+import java.time.format.DateTimeFormatter;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
I think your imports here can be organised/grouped more neatly according to their type?